app/Http/Controllers/Api/UserAvatarController.php

```diff
@@ -17,10 +18,13 @@ public function store()
             'avatar' => ['required', 'image']
         ]);
 
+        Storage::disk('public')->delete(auth()->user()->getOriginal('avatar_path'));
+
         auth()->user()->update([
```

Comment:
Since you are calling `auth()->user()` multiple times I would suggest to place it in a variable so you don't have multiple query calls.

Reply:
the user model is cached calling `auth()->user()` multiple times won't trigger another query

Reply:
A coworker told me the same indeed.
I was almost certain it did multiple queries.

Thanks for the reply.